src/components/radio/radio.ts

```diff
@@ -58,17 +58,17 @@ export default class SlRadio extends LitElement {
     const radios = this.getAllRadios();
     const checkedRadio = radios.find(radio => radio.checked);
 
-    radios.map(radio => {
-      if (radio.input) {
+    setTimeout(() => {
```

Comment:
I think the issue was that the inputs are not yet returned from `@query` since this is the first update. I can try it with the lifecycle method instead, that does make more sense. Let me know if I'm missing something else here. I'm not very familiar with lit's lifecycle and decorator behaviors.

Reply:
In that case, `this.updateComplete` should give you what you need.